modules/pod-short-term-auth-gcp-deploy-pod.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-short-term-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[discrete]
+[id="pod-short-term-auth-gcp-deploy-pod_{context}"]
+= Deploying a pod that authenticates with {gcp-wid-short}
+
+To use short-term authentication, you must create a deployment that creates a pod that triggers the webhook.
+The webhook mutates pods that the {product-title} service account creates so that they can impersonate the {gcp-short} service account.
+
+.Prerequisites
+
+* Your {gcp-short} cluster is running {product-title} version 4.17.TBD or later and uses {gcp-wid-short}.
+
+* You have created a federated {gcp-short} service account.
+
+* You have created an {product-title} service account for {gcp-short}.
+
+.Procedure
+
+. To create a pod that authenticates with {gcp-wid-short}, create a deployment YAML file similar to the following:
++
+.Sample deployment
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ubi9
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ubi9
+  template:
+    metadata:
+      labels:
+        app: ubi9
+    spec:
+      serviceAccountName: "<service_account_name>" <1>
+      containers:
+        - name: ubi
+          image: 'registry.access.redhat.com/ubi9/ubi-micro:latest'
+          command:
+            - /bin/sh
+            - '-c'
+            - |
+              sleep infinity
+----
+<1> Specify the name of the {product-title} service account.
+
+. Apply the deployment file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f deployment.yaml
+----
+
+.Verification
+
+* To verify that a pod is using short-term authentication, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -o json | jq -r '.items[0].spec.containers[0].env[] | select(.name=="GOOGLE_APPLICATION_CREDENTIALS")'
+----
++
+.Example output
+[source,terminal]
+----
+{   "name": "GOOGLE_APPLICATION_CREDENTIALS",   "value": "/var/run/secrets/workload-identity/federation.json" } 
+----
++
+The presence of the `GOOGLE_APPLICATION_CREDENTIALS` environment variable indicates a pod that authenticates with {gcp-wid-short}.
+
+* To verify additional configuration details, examine the pod specification.
+The following example pod specifications show the environment variables and volume fields that the webhook mutates.
++
+--
+.Example pod specification with the `direct` injection mode:
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: app-x-pod
+  namespace: service-a
+annotations:
+  cloud.google.com/skip-containers: "init-first,sidecar"
+  cloud.google.com/external-credentials-json: |- <1>
+    {
+      "type": "external_account",
+      "audience": "//iam.googleapis.com/projects/<project_number>/locations/global/workloadIdentityPools/on-prem-kubernetes/providers/<cluster_name>",
```

Comment:
`<cluster_name>` to be updated to `<provider>` if also done so in my prior comment.